src/openforms/forms/tests/admin/test_views.py

```diff
@@ -305,3 +308,271 @@ def test_single_import(self, m_import):
+        )
+        self.assertRedirects(
+            redirect,
+            redirect_url,
+            fetch_redirect_response=False,
+            target_status_code=302,
+        )
+
+        form_to_migrate.refresh_from_db()
+
+        self.assertEqual(form_to_migrate.payment_backend, "ogone-legacy")
+        self.assertEqual(
+            form_to_migrate.payment_backend_options, {"merchant_id": ogone_merchant.pk}
+        )
+
+    def test_form_permissions(self):
+        user = StaffUserFactory(user_permissions=[])
+        ogone_merchant = OgoneMerchantFactory()
+        form_to_migrate = FormFactory(
+            name="Ogone form 1",
+            payment_backend="ogone-legacy",
+            payment_backend_options={"merchant_id": ogone_merchant.pk},
+        )
+
+        response = self.app.post(
+            reverse("admin:forms_payment_migration"),
+            {"forms_to_migrate": [form_to_migrate.pk]},
+            user=user,
+            expect_errors=True,
+        )
+
+        self.assertEqual(response.status_code, 403)
+
+        form_to_migrate.refresh_from_db()
+
+        self.assertEqual(form_to_migrate.payment_backend, "ogone-legacy")
+        self.assertEqual(
+            form_to_migrate.payment_backend_options, {"merchant_id": ogone_merchant.pk}
+        )
+
+    def test_confirmation_page(self):
+        ogone_merchant = OgoneMerchantFactory(
+            label="Merchant Y", pspid="merchant-y", endpoint_preset=OgoneEndpoints.test
+        )
+        FormFactory(
+            name="Ogone form 1",
+            payment_backend="ogone-legacy",
+            payment_backend_options={"merchant_id": ogone_merchant.pk},
+        )
+
+        response = self.app.post(
+            reverse("admin:forms_form_changelist"),
+            {
+                "action": "migrate_to_worldline",
+                "select_across": "0",
+                "index": "0",
+                "_selected_action": "1",
+            },
+            user=self.admin_user,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+    def test_single_form(self):
+        ogone_merchant = OgoneMerchantFactory(
+            label="Merchant Y", pspid="merchant-y", endpoint_preset=OgoneEndpoints.test
+        )
+        form_to_migrate = FormFactory(
+            name="Ogone form 1",
+            payment_backend="ogone-legacy",
+            payment_backend_options={"merchant_id": ogone_merchant.pk},
+        )
+
+        response = self.app.post(
+            reverse("admin:forms_payment_migration"),
+            {"forms_to_migrate": [form_to_migrate.pk]},
+            user=self.admin_user,
+        )
+
+        self.assertRedirects(response, reverse("admin:forms_form_changelist"))
+
+        form_to_migrate.refresh_from_db()
+
+        self.assertEqual(form_to_migrate.payment_backend, "worldline")
+        self.assertEqual(
+            form_to_migrate.payment_backend_options, {"merchant": "merchant-y"}
+        )
```

Comment:
```suggestion
        self.assertRedirects(response, reverse("admin:forms_form_changelist"))
        form_to_migrate.refresh_from_db()
        self.assertEqual(form_to_migrate.payment_backend, "worldline")
        self.assertEqual(
            form_to_migrate.payment_backend_options, {"merchant": "merchant-y"}
        )
```
please maintain the visual structure of

```
arrange

act

assert
```
extra blank lines make it harder to scan for what the test setup, execution and assertions are